pubsublite-spark-sql-streaming/pom.xml

```diff
@@ -102,6 +102,21 @@
       <artifactId>jackson-core</artifactId>
       <version>2.12.0</version>
     </dependency>
+    <dependency>
```

Comment:
No idea what these dependencies are. Please remove them, they're not needed.

Reply:
Ah, the first one at least is a backport/export of AsyncLoadingCache. SGTM. Let me see how the others are used

Reply:
Yeah these are fine.